modules/ROOT/pages/authentication-authorization/index.adoc

```diff
@@ -1,24 +1,108 @@
+
+[[authorization-overview]]
+== Authorization
+
+Authorization is the process of determining whether a user is allowed to perform a specific action.
+Authorization is managed using role-based access control (_RBAC_).
+RBAC is a method of restricting access to authorized users.
+It is a way of assigning privileges to roles that are then assigned to users.
+This simplifies user management, as permissions are assigned to roles rather than to individual users.
+The roles are defined in terms of their underlying _privileges_, and they can be modified by adding or removing these access rights using the Cypher commands described in this chapter.
+
+Neo4j provides a set of xref:authentication-authorization/built-in-roles.adoc[built-in roles] and also allows you to create custom roles with specific privileges.
+You can also use the _sub-graph_ access control, through which read access to the graph can be limited to specific combinations of labels, relationship types, and properties.
+
 [NOTE]
 --
-The functionality described in this section is applicable to Enterprise Edition.
+The functionality described in these pages applies to Enterprise Edition.
 A limited set of user management functions are also available in Community Edition.
-xref:authentication-authorization/built-in-roles.adoc#auth-built-in-roles-overview[Native roles overview] gives a quick overview of these.
---
\ No newline at end of file
+xref:authentication-authorization/built-in-roles.adoc#auth-built-in-roles-overview[Built-in roles capabilities] gives a quick overview of these.
+--
+
+The Neo4j security model is stored in the system graph, which is maintained in the xref:database-administration/index.adoc#manage-databases-system[`system` database].
+All administrative commands need to be executed against it.
+When connected to the DBMS over xref:configuration/connectors.adoc[], administrative commands are automatically routed to the `system` database.
+
+[[auth-terminology]]
+== Terminology
+
+The following terms are relevant to role-based access control within Neo4j:
+
+[.compact]
+[[term-active-user]]active user::
+A user who is active within the system and can perform actions prescribed by any assigned roles on the data.
+This is in contrast to a suspended user.
+
+[[term-administrator]]administrator::
+This is a user who has been assigned the admin role.
+
+[[term-current-user]]current user::
+This is the currently logged-in user invoking the commands.
+
+[[term-password-policy]]password policy::
+The password policy is a set of rules about what makes up a valid password.
+For Neo4j, the following rules apply:
+* The password cannot be an empty string.
+* When changing passwords, the new password cannot be the same as the previous password.
+
+[[term-role]]role::
+This is a collection of privileges for specific actions permitted on the data.
+Users can be assigned different roles, and a user can have multiple roles.
```

Comment:
doesn't this sentence just say the same thing twice? I assume one part should imply users can have multiple roles and the other that roles can be assigned to multiple users? but it feels like they just say users can have multiple roles twice (or perhaps that different users don't need to have the same roles?)